sql/src/main/java/org/apache/druid/sql/calcite/planner/CalcitePlanner.java

```diff
@@ -502,4 +505,25 @@ void from(CalcitePlanner planner)
                                          + " to " + this);
     }
   }
+
+  /**
+   * Define some tool members and methods for hints.
+   */
+  private static class HintTools
+  {
+    static final HintStrategyTable HINT_STRATEGY_TABLE = createHintStrategies();
+
+    /**
+     * Creates hint strategies.
+     *
+     * @return HintStrategyTable instance
+     */
+    private static HintStrategyTable createHintStrategies()
+    {
+      return HintStrategyTable.builder()
+                              .hintStrategy(JoinHint.BROADCAST.name(), HintPredicates.JOIN)
```

Comment:
is the usage of `name()` intended here?
because I think that will return `BROADCAST` the "name" of the enum key and not the `id`!

Reply:
note: this method returning `HintStrategyTable` could live inside `DruidHint` if that's created

Reply:
Moved to DruidHint